components/LogWorkout/LogWorkout.jsx

```diff
@@ -111,9 +112,9 @@ const LogWorkout = (props) => {
         exerciseName: exercise.name,
         sets: exercise.sets.map((set) => {
           const parsedSet = {
-            weight: set.weight || set.prevWeight,
+            weight: (set.prevWeight == 'n/a' && set.weight  == '') ? null : parseInt(set.weight) || parseInt(set.prevWeight),
```

Comment:
Use triple equals in javascript please.

Would something like this work better:
`parseInt(set.weight) || set.prevWeight === 'n/a' ? null : parseInt(set.prevWeight)`

This way we only check that prevWeight is n/a when we have to. Also I'm p sure NaN is falsey so you don't need to worry about set.weight being an empty string.

Reply:
ya lol idk why I thought oof the logic like that haha

Reply:
actually I tried your way and it doesnt work because I have to check  set.weight check for ' ' otherwise if I put a weight it sends null